lib/install/validate/config_to_data.go

```diff
@@ -0,0 +1,318 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
```

Comment:
This is a large new file, but it doesn't look like you added any unit tests?  Can you please add at least some unit tests for this new file?